src/release/breaking-changes/tab-alignment.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Customizing tabs alignment using the new TabBar.tabAlignment property
+description: Introducing the TabBar.tabAlignment property.
+---
+
+## Summary
+
+Using `TabBar.tabAlignment` to customize the alignment of tabs in a `TabBar`.
+
+## Context
+
+The `TabBar.tabAlignment` property is introduced to customize the alignment of tabs in a Material 3 `TabBar`. The `TabAlignment` enum has the following values:
+
+* `TabAlignment.start`: Aligns the tabs to the start of the scrollable `TabBar`.
+* `TabAlignment.startOffset`: Aligns the tabs to the start of the scrollable `TabBar` with an offset of `52.0` pixels.
+* `TabAlignment.center`: Aligns the tabs to the center of the `TabBar`.
+* `TabAlignment.fill`: Aligns the tabs to the start and stretches the tabs to fill the fixed `TabBar`.
```

Comment:
Should these descriptions also mention which ones work with `isScrollable` and which ones don't?